Sources/HummingbirdLambda/HBLambdaRequestContext.swift

```diff
@@ -0,0 +1,10 @@
+import Logging
+import NIOCore
+
+/// A Request Context that contains the Event that triggered the Lambda
+public protocol HBLambdaRequestContext<Event>: HBBaseRequestContext {
+    /// The type of event that can trigger the Lambda
+    associatedtype Event
+
+    init(_ event: Event, applicationContext: HBApplicationContext, eventLoop: EventLoop, allocator: ByteBufferAllocator, logger: Logger)
```

Comment:
Why don't you just pass in the whole `LambdaContext`. Maybe someone else will want to extract more information from it

Reply:
No particular reason actually